assets/airports/ulli.json

```diff
@@ -0,0 +1,2695 @@
+            "draw": [
+                ["GENPA*", "LI753", "LI754", "LI749", "LI750", "LI751", "LI752", "OBARI"]
+            ]
+        },
+        "KOLET1A": {
+            "icao": "KOLET1A",
+            "name": "Kolet One Alpha",
+            "entryPoints": {
+                "KOLET": [["KOLET", "A150-"]]
+            },
+            "body": [
+                ["LI730", "A60+|A150-"],
+                ["LI724", "A50|S225"],
+                ["LI725", "A50|S225"],
+                ["LI726", "S225"],
+                ["LI727", "S225"],
+                ["LI728", "S225"],
+                ["LI729", "A50|S225"],
+                ["DIKOM", "A25+|A35-"]
+            ],
+            "rwy": {
+                "ULLI10L": [["LEPTO", "A25"]],
+                "ULLI10R": [["GEBKA", "A25"]]
+            },
+            "draw": [
+                ["KOLET*", "LI730", "LI724", "LI725", "LI726", "LI727", "LI728", "LI729", "DIKOM"]
+            ]
+        },
+        "KOLET1B": {
+            "icao": "KOLET1B",
+            "name": "Kolet One Bravo",
+            "entryPoints": {
+                "KOLET": [["KOLET", "A250-"]]
+            },
+            "body": [
+                ["LI755", "A60+|A250-|S225-"],
+                ["LI746", "A60"],
+                ["LI747", "A50+|A60-|S225-"],
+                ["LI748", "A50|S225"],
+                ["LI749", "S225"],
+                ["LI750", "S225"],
+                ["LI751", "S225"],
+                ["LI752", "A50|S225"],
+                ["OBARI", "A25+|A35-"]
+            ],
+            "rwy": {
+                "ULLI28L": [["BIPRI", "A25"]],
+                "ULLI28R": [["MADID", "A25"]]
+            },
+            "draw": [
+                ["KOLET*", "LI755", "LI746", "LI747", "LI748", "LI749", "LI750", "LI751", "LI752", "OBARI"]
+            ]
+        },
+        "OBANA1A": {
+            "icao": "OBANA1A",
+            "name": "Obana One Alpha",
+            "entryPoints": {
+                "OBANA": [["OBANA", "A150-"]]
+            },
+            "body": [
+                ["LI731", "A60+|A150-|S270-"],
+                ["LI725", "A50|S225"],
+                ["LI726", "S225"],
+                ["LI727", "S225"],
+                ["LI728", "S225"],
+                ["LI729", "A50|S225"],
+                ["DIKOM", "A25+|A35-"]
+            ],
+            "rwy": {
+                "ULLI10L": [["LEPTO", "A25"]],
+                "ULLI10R": [["GEBKA", "A25"]]
+            },
+            "draw": [
+                ["OBANA*", "LI731", "LI725", "LI726", "LI727", "LI728", "LI729", "DIKOM"]
+            ]
+        },
+        "OBANA1B": {
+            "icao": "OBANA1B",
+            "name": "Obana One Bravo",
+            "entryPoints": {
+                "OBANA": [["OBANA", "A250-"]]
+            },
+            "body": [
+                ["LI744", "A70+|A140-|S225-"],
+                ["LI745", "A70"],
+                ["LI746", "A60"],
+                ["LI747", "A50+|A60-|S225-"],
```

Comment:
The *altitude* restrictions at LI747 are for holding only, not part of the OBANA1B arrival